### ISSUE TITLE
Can't set source directory 

### ISSUE THREAD

**Issue by rjclancy** [renatoathaydes/spock-reports#255]
I'm trying to target a specific test directory i.e sec/main/groovy/.../integration that I would like to generate my reports but overriding this in my build gradle doesn't work

test System Property: com.athaydes.spockframework.report.testSourceRoots=src/test/groovy

Is this the correct approach?

**Comment by renatoathaydes** [comment]
Yes, that should work... if not, can you try the `SpockConfig.groovy` file as shown [here](https://github.com/renatoathaydes/spock-reports#using-spockconfiggroovy)?

**Comment by renatoathaydes** [comment]
Notice that this property is only used to include the source code in the reports:

```properties
# Set the root location of the Spock test source code (only used if showCodeBlocks is 'true')
com.athaydes.spockframework.report.testSourceRoots=src/test/groovy
```

Is this your problem? Or are you trying to get Spock to find your actual tests? If that's the case, then you need to change your groovy config.

**Comment by rjclancy** [comment]
Thanks @renatoathaydes. Yup, I just want to generate reports for tests within my src/main/test/integration dir exclude everything else in src/main/test/..

**Comment by renatoathaydes** [comment]
So, tests are executed but no reports are generated?

**Comment by rjclancy** [comment]
> So, tests are executed but no reports are generated?

All my tests are executed. I just would like to generate the Spock report for a specific directory.

src/main/groovy/service: don't care

src/main/groovy/integration: do care

**Comment by renatoathaydes** [comment]
spock-reports is called by Spock for each test that runs, there's no way to exclude tests from reports.
